### PR TITLE
Improve `HeadingBlock` preview skeleton

### PR DESCRIPTION
**Pull request by SebiVPS**
> Ich würd bei Eyebrow kein PreviewSkeleton machen, weil sie sowieso optional ist. Bei Heading macht’s noch mehr Sinn.
Dann gibt’s für den HeadlineBlock auch nur einen Placeholder, was ich besser finde.

- Remove preview skeleton for eyebrow.
- Improve representation of preview skeleton of the headline

---

## PR Checklist

-   [x] Provide screenshots/screencasts if the change contains visual changes

<details>
    <summary>Screenshots/screencasts</summary>
before:
Screenshot 2024-07-12 at 08 46 07
after:
Screenshot 2024-07-12 at 08 45 38
</details>

### CODE REVIEW

**Review by jamesricky — APPROVED** [review]

File: site/src/common/blocks/HeadingBlock.tsx [hunk 1]
```diff
@@ -76,10 +76,21 @@ export const HeadingBlock = withPreview(
...
+                    </Typography>
+                )}
+                <PreviewSkeleton
+                    hasContent={hasRichTextBlockContent(headline)}
+                    title={
+                        <HeadlineSkeleton variant={"h550"} component={"span"}>
+                            Headline lorem Ipsum
+                        </HeadlineSkeleton>
+                    }
+                >
```
Comment (thomasdax98) [inline site/src/common/blocks/HeadingBlock.tsx:91]: Why do we need this? Isn't the PreviewSkeleton already in `RichTextBlock`

Comment (SebiVPS) [reply site/src/common/blocks/HeadingBlock.tsx:91]: Yes there is already one. But i do not see an easy way to adapt this one to look like expected. You have a suggestion?

Comment (thomasdax98) [reply site/src/common/blocks/HeadingBlock.tsx:91]: Ah okay, now I understand. I guess it's okay then

File: site/src/common/blocks/HeadingBlock.tsx [hunk 2]
```diff
@@ -76,10 +76,21 @@ export const HeadingBlock = withPreview(
...
+                {hasRichTextBlockContent(eyebrow) && (
+                    <Typography variant={"h400"} component={"h5"} bottomSpacing>
+                        <RichTextBlock data={eyebrow} renderers={eyebrowRenderers} />
+                    </Typography>
+                )}
+                <PreviewSkeleton
+                    hasContent={hasRichTextBlockContent(headline)}
+                    title={
+                        <HeadlineSkeleton variant={"h550"} component={"span"}>
+                            Headline lorem Ipsum
```
Comment (thomasdax98) [inline site/src/common/blocks/HeadingBlock.tsx:88]: But I wouldn't add lorem ipsum here

```suggestion
                            Headline
```

**Review by thomasdax98 — APPROVED** [review]

**Review by jamesricky — APPROVED** [review]

**Review by jamesricky — APPROVED** [review]

**Review by thomasdax98 — APPROVED** [review]

**Pull request merged**